drivers/media/v4l2-core/v4l2-ctrls.c

```diff
@@ -2209,16 +2209,25 @@ static void new_to_req(struct v4l2_ctrl_ref *ref)
 	if (!ref)
 		return;
 	ptr_to_ptr(ref->ctrl, ref->ctrl->p_new, ref->p_req);
-	ref->req = ref;
```

Comment:
Commit text again. Looks like https://github.com/torvalds/linux/commit/ac34b79da14d67a9b494f6125186becbd067e225

Seeing as it has 
```Cc: <stable@vger.kernel.org>      # for v5.9 and up```
I'm surprised it hasn't made it back to 5.10 already.

Reply:
Again a cherry-pick from Hans' tree from before being upstreamed. I'm going to guess it will turn up soon.